CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 
-cmake_minimum_required(VERSION 3.2.1)
+cmake_minimum_required(VERSION 3.17.0)
 cmake_policy(VERSION 3.2.1)
 
 if(NOT CMAKE_VERSION MATCHES "ReactOS")
```

Comment:
Is this check still true/needed? For non-MSVC?

Reply:
https://chat.reactos.org/reactos/pl/7weqy76ocfn55p9m6snmpzcddc

Reply:
> https://chat.reactos.org/reactos/pl/7weqy76ocfn55p9m6snmpzcddc

Hum... It is written there that:
* (RosBE 2.2) `3.17-ReactOS` custom CMake has no special features, except being compiled to run on XP too.
* Want to keep the warning, with regard to possible 3rd party custom CMake.

Then, I would suggest to:
* Rephrase it, at least to add that official CMake is fully fine too.
* Update GitHub Actions MSVC builds to use RosBE CMake, not to trigger the (confusing) warning.